spec/fractal_spec.rb

```diff
@@ -1,7 +1,9 @@
 require 'fractal'
 
-describe Fractal::generate do
-  it 'will return fractal' do
-  	expect(Fractal.generate).to eq "Kirill"
+describe Fractal do
+  describe '#generate' do
```

Comment:
I was trying to use special syntax for Ruby on Rails. Correct way is to split class and method, like here: http://www.betterspecs.org/#data